image-cropper/src/app/cropper/canvas/tool.rs

```diff
@@ -123,103 +125,103 @@ fn render_zoom_icon() -> RenderingTree {
 
 fn get_poly_selection_path() -> PathBuilder {
     PathBuilder::new()
-        .move_to(1.0, 1.0)
-        .line_to(0.8, 0.8)
-        .line_to(0.2, 0.6)
-        .line_to(0.0, 0.2)
-        .line_to(0.5, 0.0)
-        .line_to(0.7, 0.2)
-        .line_to(0.8, 0.8)
+        .move_to(px(1.0), px(1.0))
+        .line_to(px(0.8), px(0.8))
+        .line_to(px(0.2), px(0.6))
+        .line_to(px(0.0), px(0.2))
+        .line_to(px(0.5), px(0.0))
+        .line_to(px(0.7), px(0.2))
+        .line_to(px(0.8), px(0.8))
         .close()
 }
 
 fn get_hand_path() -> PathBuilder {
     PathBuilder::new()
         .arc_to(
-            &LtrbRect {
-                left: 0.0625,
-                top: 0.5625,
-                right: 0.1875,
-                bottom: 0.6875,
+            Rect::Ltrb {
+                left: px(0.0625),
+                top: px(0.5625),
+                right: px(0.1875),
+                bottom: px(0.6875),
             },
-            2.251705961447832,
-            3.141592653589793,
+            Angle::Radian(2.251705961447832),
```

Comment:
I have no idea what this value is :(

Reply:
That may be 
![image](https://user-images.githubusercontent.com/38313680/177440550-7404e0ce-95d3-4014-a892-bccdb6f05711.png)

Reply:
![image](https://user-images.githubusercontent.com/3580430/177482408-6b7600fd-222a-4568-9211-224d7a805c99.png)